LANDSCAPE/COMPARISONS/ARGOCD-FLUX/index.md

```diff
@@ -0,0 +1,275 @@
+# Continuous Deployment on Kubernetes: What’s the Difference Between Argo CD and FluxCD
```

Comment:
```suggestion
# Continuous Deployment on Kubernetes: What’s the Difference Between Argo CD and Flux CD
```

Reply:
Flux is just called "Flux".

Reply:
Thanks @squaremo

Reply:
Thank you for following through on my rather terse comment :pray:

(Context: Flux's github org and website domain use "fluxcd" ("flux" was gone already), which has caused ongoing confusion, even amongst its maintainers...)